submissions/dikun/html-css-popup/index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&family=Space+Mono&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">  
+    <title>Popup</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="header__navigation">
+            <ul class="navigation__items">
+                <li class="navigation__item">
+                    <a href="#" class="navigation__link">Mail</a>
+                </li>
+                <li class="navigation__item">
+                    <a href="#" class="navigation__link">Images</a>
+                </li>
+                <li class="navigation__item popup">
+                    <input type="checkbox" id="check" class="input__popup">
```

Comment:
Not fixed yet.
Your input has class `input_popup` which means this is a BEM element of the parent BEM block with class `input`. I can't see BEM block with class `.input`)

Reply:
Add `input` class.